workspace/Foo/app.json

```diff
@@ -103,7 +103,8 @@
         },
         "resources": {
             "path": "${build.id}/resources",
-            "shared": "resources"
+            "shared": "resources",
+            "_resources": "_resources"
```

Comment:
This resource pool is the one that will contain the files under "_resources". The right side is an arbitrary identifier and the left side is a path relative to the build output. For consistency, I named them the same.